R/helper_starprep.R

```diff
@@ -249,8 +249,15 @@ starprep <- function(...,
                      se_type = NULL,
                      clusters = NULL,
                      alpha = 0.05) {
+
+  fits <- list(...)
+
+  gave_list <- length(fits) == 1 & is.list(fits[[1]])
```

Comment:
I think this code is probably correct, but unclear - I would double check that `is.list` is false for lm objects - you don't want to break the case where only one model is provided.

    is.list(lm(extra~group, sleep))

Also while you are in here, can you change the `class(x)[1] == "lm"` below to use `inherits`.

Reply:
oof, good point. will update now